### DISCUSSION TITLE
Add flextv.co.kr

### DISCUSSION THREAD

**Discussion by orirenato03** [streamlink/streamlink#5836] [Q&A]
### Description

Please add flextv.co.kr to the supported websites, lot of korean stream moves there after twitch shut down.

**Comment by bastimeyer** [comment]
- https://github.com/streamlink/streamlink/blob/master/.github/DISCUSSION_TEMPLATE/q-a.yml
- https://github.com/streamlink/streamlink/issues?q=flextv
